src/jobs/push-to-app-collection.yaml

```diff
@@ -0,0 +1,19 @@
+executor: architect
+parameters:
+  app_name:
+    description: "Name of the application"
+    type: "string"
+  app_catalog:
+    description: "Name of the catalog CR where the application belongs"
+    type: "string"
+    default: "control-plane-catalog"
+  app_collection_repo:
+    description: "Name of the app collection repository"
+    type: "string"
+steps:
+  - checkout
+  - prepare-catalogbot-git-ssh
```

Comment:
We need a good readme also with information that we need env variable holding catalogbot user credentials.

Reply:
updated README. PTALA